packages/common/core-utils/src/assert.ts

```diff
@@ -5,13 +5,25 @@
 
 /**
  * A browser friendly assert library.
```

Comment:
Nit: while we're making changes here, this comment doesn't describe what the function _does_. Might be nice to update the summary here to something like:
```suggestion
 * Asserts the provided condition.
```

And update the remarks to note that this exists as a browser-friendly alternative to Node's assert, etc.